docs/source/maintenance.rst

```diff
@@ -91,39 +93,41 @@ Upon adding GMRES, the docs/source/modules/linalg/solvers.rst file has to be cha
    .. autoclass:: psydac.linalg.solvers.ConjugateGradient
       :members:
       
-   # five further classes later, add:
+   -----------------------------------             # five further classes later, add:
    
-   .. _gmres:						# this line is new
+   .. _gmres:						                     # this line is new
    
-   GMRES						# this line is new
-   -----						# this line is new
+   GMRES						                           # this line is new
+   -----						                           # this line is new
    
    .. autoclass:: psydac.linalg.solvers.GMRES		# this line is new
-      :members:						# this line is new
+      :members:						                  # this line is new
```

Comment:
Are the last changes necessary? If so why?

Reply:
No, I thought the indentation was wrong, but it has actually already been correct.
09d7d3ce32159da9d13294910b7ee016a8b8d579